logistic_requisition/model/logistic_requisition.py

```diff
@@ -1245,3 +1312,27 @@ def onchange_selected_bid_id(self):
             purchase = self.selected_bid_id
             supplier_partner_id = purchase.partner_id.id
         self.supplier_partner_id = supplier_partner_id
+
+    @api.onchange('sourcing_method')
+    def onchange_sourcing_method(self):
+        """ Set `price_is` to fixed when reusing a bid
+        """
+        self.price_is = ('fixed' if self.sourcing_method == 'reuse_bid'
+                         else False)
```

Comment:
@lepistone This field needs to be set by user with other sourcing_methods
I want to empty it when we leave "reuse_bid" method. As I can't know what was previous value, I prefer to empty it in other case. It is a 2 options selection. That shouldn't be an overhead.